src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
```

Comment:
- [ ] I will strongly recommend you fix your linter errors instead of disabling them as linters ensure you write clean and readable code 👍🏽

Reply:
KIndly implement the suggested changes. These is `required changes` and not `approved `